XCUITests/FxScreenGraph.swift

```diff
@@ -452,6 +465,35 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
         screenState.backAction = navigationControllerBackAction
     }
 
+    map.addScreenState(FxASigninScreen) { screenState in
+        screenState.backAction = navigationControllerBackAction
+
+        screenState.gesture(forAction: Action.TypeInvalidEmail) { userState in
+            app.webViews.textFields["Email"].tap()
+            typeInField(userState.fxaWrongEmail)
+        }
+        screenState.gesture(forAction: Action.TypeCorrectEmail) { userState in
+            app.webViews.textFields["Email"].tap()
+            typeInField(userState.fxaRightEmail)
+        }
+        screenState.gesture(forAction: Action.TypeInvalidPassword) { userState in
+            app.webViews.secureTextFields["Password"].tap()
+            typeInField(userState.fxaWrongPassword)
+        }
+        screenState.gesture(forAction: Action.TypeIncorrectPassword) { userState in
+            app.webViews.secureTextFields["Password"].tap()
+            typeInField(userState.fxaRightPassword)
+        }
+        screenState.gesture(forAction: Action.TapOnSignInButton) { userState in
```

Comment:
Where should this transition to?

Shouldn't this be just another screen state, rather than an action?

Reply:
So, if user and password are ok, this should transition to FxSignedin Screen. 
But here we are only testing error cases and cannot have an E2E test yet to reach to that screen, that's why I did not add the transitionTo that screen